### PR TITLE
My 2 cents

### PR DESCRIPTION
**Pull request by lyrixx**
Many small fix, see commits.
Refs: #2 BTW.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,10 +22,10 @@ $ symfony-upgrade-fixer fix /path/to/code --dry-run
 | Name  | Description |
 | ----  | ----------- |
 | property_access | Renamed PropertyAccess::getPropertyAccessor to PropertyAccess::createPropertyAccessor. |
+| inherit_data_aware_iterator | The class VirtualFormAwareIterator was renamed to InheritDataAwareIterator. |
```
Comment (umpirsky) [inline README.md]: We only change `README.tml`, then generate with `./symfony-upgrade-fixer readme > README.md`.

File: box.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
+{
+    "directories": ["src/"],
+    "finder": [
+        {
+            "name": "*.php",
+            "exclude": [
+                ".gitignore",
+                ".md",
```
Comment (umpirsky) [inline box.json]: `.tpl`

File: composer.json [hunk 3]
```diff
@@ -16,9 +16,6 @@
     "require-dev": {
         "phpunit/phpunit": "^4.8"
     },
-    "config": {
```
Comment (umpirsky) [inline composer.json]: Why removed?

**Comment by umpirsky** [comment]
@lyrixx Very cool, thanks for PR, I really appreciate your contribution! :+1:

File: composer.json (hunk 3 above)
Comment (lyrixx) [inline composer.json]: This is the default value? And it does not change anything.

File: README.md (hunk 1 above)
Comment (lyrixx) [inline README.md]: I know. But I got a diff. So I updated it.

File: composer.json (hunk 3 above)
Comment (umpirsky) [inline composer.json]: Oh, ok, didn't know  that. :)

File: README.md (hunk 1 above)
Comment (umpirsky) [inline README.md]: Ah, because iterator is changed, ok, cool.

File: composer.json [hunk 4]
```diff
@@ -34,5 +31,5 @@
             "dev-master": "0.1-dev"
         }
     },
-    "bin": ["symfony-upgrade-fixer"]
+    "bin": ["bin/symfony-upgrade-fixer"]
```
Comment (umpirsky) [inline composer.json]: @lyrixx Why is it moved to bin? This project extends [PHP-CS-Fixer](https://github.com/FriendsOfPHP/PHP-CS-Fixer), I tend to keep same structure to make contribution easier.

Comment (lyrixx) [inline composer.json]: To respect standard. Binary should be in bin dir :-)

Comment (umpirsky) [inline composer.json]: OK, I trust you. That will not make things harder for sure. ;)

**Comment by umpirsky** [comment]
@lyrixx Can you please rebase?

File: README.md (hunk 1 above)
Comment (lyrixx) [inline README.md]: Actually, I think it comes from the finder. Not the alpha sorted ;) so it will be stable.

**Comment by lyrixx** [comment]
Rebased.

**Comment by umpirsky** [comment]
Thanks @lyrixx, I'm really happy you contributed! :+1:

**Pull request merged**